packages/react-core/src/components/Menu/examples/Menu.md

```diff
@@ -119,3 +118,8 @@ To render an initially drilled in menu, the `menuDrilledIn`, `drilldownPath`, an
 
 ```ts file="MenuWithViewMore.tsx"
 ```
+
+### Danger menu item
```

Comment:
@tlabaj oh, I made a comment in the core PR to move it after the "view more" and "loading" examples, since those seem like similar variations, but I missed that it wasn't made before merging. WDYT? https://github.com/patternfly/patternfly/pull/5102#pullrequestreview-1116199160